packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -454,6 +454,30 @@ export class DataStores implements IDisposable {
         return builder.getGCData();
     }
 
+    /**
+     * After GC has run, called to notify this Container's data stores of routes that are used in it.
+     * @param usedRoutes - The routes that are used in all data stores in this Container.
+     */
+    public updateUsedRoutes(usedRoutes: string[]) {
```

Comment:
This is the code I was looking for to complete the picture. I am curious if this is consistent do you want to extract it? If not, i.e. if you are thinking the logic for mapping GC routes to data store contexts, etc. may be customizable, then it makes more sense to keep it here.

Reply:
Well, I am not 100% sure actually. Once we have GC at DDS layer, it will become clear if we can re-use this code. From what I can tell today, it should be re-usable but I would rather wait until that work is done to extract it.